src/ActionBar/Action.js

```diff
@@ -19,6 +19,8 @@ import Tooltip from '../Tooltip';
 import { ActionBarContext } from './ActionBar';
 
 const Action = ({ children, icon, ...other }) => {
+  const actionBarContext = useContext(ActionBarContext);
+
   const getAction = ({ action, collapsed }) => {
```

Comment:
This seems like an unnecessary abstraction if it's just keying off of `collapsed`, thoughts on refactoring?